pkg/cache/cache_test.go

```diff
@@ -229,6 +232,9 @@ func CacheTest(createCacheFunc func(config *rest.Config, opts cache.Options) (ca
 				})
 
 				It("should be able to list objects with GVK populated", func() {
+					if opts.UnsafeDisableCacheDeepCopy {
+						Skip("skip for UnsafeDisableCacheDeepCopy enabled")
+					}
```

Comment:
I'm not sure what's better. This use of `Skip` or wrapping the entire `It` in an if statement to prevent it from being declared at all in these cases.

Based on ginkgo docs:
> By default, Ginkgo will print out a description for each skipped spec. You can suppress this by setting the --noisySkippings=false flag.

That makes me lean toward an outer if statement around the `It`. Since we'll get test output will _always_ contain these intentionally skipped tests, and we likely _do_ want to keep noisy skippings enabled to signal.

Anyone else have thoughts here about what's more idiomatic or preferred?

Reply:
Yep, I have moved the `It` into an if statement and there is no more `skip` in ginkgo test. I think this is better.